defaults/azure_config.yaml

```diff
@@ -20,9 +20,10 @@ root_disk_size_loader: 30  # GB, Increase loader disk in order to have extra spa
 loader_swap_size: 10240  #10GB SWAP space
 azure_image_username: 'scyllaadm'
 # used prepared centos7 AMI for loader
-ami_loader_user: 'centos'
+ami_loader_user: 'ubuntu'
+scylla_repo_loader: 'https://s3.amazonaws.com/downloads.scylladb.com/deb/debian/scylla-4.6-buster.list'
```

Comment:
Maybe 5.0?

Reply:
I'm guessing that's what we have in AWS/GCE right now.
it's a note to myself to remove this option when moving to running docker for loaders.

Reply:
yes, I took it from AWS/GCE